server/api/orders.js

```diff
@@ -0,0 +1,24 @@
+const router = require('express').Router()
+const {Order} = require('../db/models')
+
+module.exports = router
+
+// GET /api/orders
+router.get('/', async (req, res, next) => {
+  try {
+    const hues = await Order.findAll()
```

Comment:
Slight change — would it make more sense to name this constant orders and res.json(orders)?